packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -1810,12 +1810,14 @@ class _CupertinoDialogActionsRenderWidget extends MultiChildRenderObjectWidget {
       ..dialogWidth = _isActionSheet
           ? null
           : _isInAccessibilityMode(context)
-            ? _kAccessibilityCupertinoDialogWidth
-            : _kCupertinoDialogWidth
+              ? _kAccessibilityCupertinoDialogWidth
+              : _kCupertinoDialogWidth
       ..dividerThickness = _dividerThickness
-      ..dialogColor = CupertinoDynamicColor.resolve(_isActionSheet ? _kActionSheetBackgroundColor : _kDialogColor, context)
+      ..dialogColor =
+          CupertinoDynamicColor.resolve(_isActionSheet ? _kActionSheetBackgroundColor : _kDialogColor, context)
       ..dialogPressedColor = CupertinoDynamicColor.resolve(_kPressedColor, context)
-      ..dividerColor = CupertinoDynamicColor.resolve(_isActionSheet ? _kActionSheetButtonDividerColor : CupertinoColors.separator, context)
+      ..dividerColor = CupertinoDynamicColor.resolve(
+          _isActionSheet ? _kActionSheetButtonDividerColor : CupertinoColors.separator, context)
```

Comment:
This looks pretty OK. Expressions are moved onto newlines as needed and nothing is split up weird. indentation is consistent